R/view.R

```diff
@@ -24,28 +24,55 @@
 view <- function(x, title = NULL, ..., n = NULL) {
   check_dots_empty()
 
-  if (!interactive()) return(invisible(x))
+  if (!rlang::is_interactive()) {
```

Comment:
Do we need the qualification here?

```suggestion
  if (!is_interactive()) {
```